lib/gooddata/models/metadata/report.rb

```diff
@@ -174,6 +174,16 @@ def export(format, options = {})
       client.poll_on_code(result1['uri'], options.merge(process: false))
     end
 
+    # Exports a report too large to be computed on the UI. It executes in raw form.
+    # Saves the result into file.
+    #
+    # @param filename [String] Filename to save into
+    # @return [Net::HTTPResponse] Returns HTTP status
+    def export_raw(filename, options = {})
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used.